databricks_cli/__init__.py

```diff
@@ -24,6 +24,22 @@
 # pylint:disable=import-error
 # pylint:disable=bare-except
 
+import warnings
+
+
+def issue_deprecation_warning():
+    # Don't print deprecation warning when running the CLI itself.
+    import sys
+    import os
+    if sys.argv and os.path.basename(sys.argv[0]) == 'databricks':
+        return
```

Comment:
Could we add a check here to only fire the warning once?

I ran `python3 -m databricks_cli.cli` and got a spew of a dozen identical warnings.